### PR TITLE
Move hpp-gui core code to gepetto-viewer-corba

### PR DESCRIPTION
**Pull request by heidydallard**

### CODE REVIEW

**Comment by jmirabel** [comment]
Please, rebase your changes (https://github.com/humanoid-path-planner/gepetto-viewer-corba/pull/22/commits/94a0d832f8fb1afcb0585659dca845571ab68ca2) onto the devel branch before. This is a big change and I prefer to make it as atomic as possible.

File: include/gepetto/gui/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#
```
Comment (jmirabel) [inline include/gepetto/gui/CMakeLists.txt]: This file can be deleted.

File: include/gepetto/gui/hpp-gui.in [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```
Comment (jmirabel) [inline include/gepetto/gui/hpp-gui.in]: This file should not be here.

File: src/gui/CMakeLists.txt.user [hunk 3]
```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jmirabel) [inline src/gui/CMakeLists.txt.user]: Remove this file.

File: include/gepetto/gui/main.cc.in [hunk 4]
```diff
@@ -0,0 +1,91 @@
+#include <QtGui/QApplication>
```
Comment (jmirabel) [inline include/gepetto/gui/main.cc.in]: This file should be removed.

File: src/gui/main.cc.in [hunk 5]
```diff
@@ -0,0 +1,91 @@
+#include <QtGui/QApplication>
```
Comment (jmirabel) [inline src/gui/main.cc.in]: Same thing here.

**Comment by heidydallard** [comment]
I think it is ok right now.

**Pull request merged**